README.md

```diff
@@ -53,6 +53,30 @@ val result = Yaml.default.encodeToString(Team.serializer(), input)
 println(result)
 ```
 
+### Parsing into YamlNode
+
+It is possible to parse a string or an InputStream directly into a YamlNode, for example
+the following code prints `Cindy`.
+```kotlin
+val input = """
+        leader: Amy
+        members:
+          - Bob
+          - Cindy
+          - Dan
+    """.trimIndent()
+
+val result = Yaml.default.parseToYamlNode(input)
+
+println(
+    result
+        .yamlMap.get<YamlNode>("members")!!
```

Comment:
If the `get` call is changed to `get<YamlList>()`, does that remove the need for the `yamlList` call?

Reply:
Changed it to `get<YamlList>`.